test/e2e/disabling_copied_csv_e2e_test.go

```diff
@@ -260,3 +279,24 @@ var _ = Describe("Disabling copied CSVs", func() {
 		})
 	})
 })
+
+func getProtectedCopiedCSVNamespaces(protectedCopiedCSVNamespaces map[string]struct{}) error {
+	var olmDeployment appsv1.Deployment
+	if err := ctx.Ctx().Client().Get(context.TODO(), apitypes.NamespacedName{Name: "olm-operator", Namespace: operatorNamespace}, &olmDeployment); err != nil {
```

Comment:
nit/not blocking: avoid hardcoding the context and namespaced name fields and pass those as parameters to this function instead?